### ISSUE TITLE
More complete Placement editor

### ISSUE THREAD

**Issue by amyjko** [wordplaydev/wordplay#172]
## What's the problem?

Editing Placement streams should be easy.

## What's the idea?

Placement streams can currently only be edited in minimal ways. All inputs on a Placement() stream should be editable in the palette.

## Who benefits?

Anyone configuring a Placement.

## Design

See [approved design](https://github.com/wordplaydev/wordplay/issues/172#issuecomment-2723355932).

**Comment by Yinn7** [comment]
Hi @amyjko , can I be assigned for this issue?

**Comment by amyjko** [comment]
Assigned! Let me know how I can help. A simple design sketch for this would suffice.

**Comment by Yinn7** [comment]
Hi @amyjko , here's the design sketch for the placement setting

![Image](https://github.com/user-attachments/assets/d088f167-5648-4c05-8721-e64d09268261)

**Comment by amyjko** [comment]
This design looks great. The only tweak is that there's no need for a "Set" button. All of the palette controls immediately update upon edit. Delete that an then this design is approved.

**Comment by Yinn7** [comment]
The update sketch

![Image](https://github.com/user-attachments/assets/db7efa61-cd05-46e0-86f0-70808175a26f)

**Comment by amyjko** [comment]
Approved! Thank you for your contribution to the community.

**Comment by annieli555** [comment]
@amyjko - Think the assignee needs to be removed here!
